appinventor/appengine/war/index.jsp

```diff
@@ -1,3 +1,20 @@
+<%@page import="com.google.appinventor.server.Server,com.google.appinventor.common.version.AppInventorFeatures" %>
+<%
+   if (request.getScheme().equals("http") && Server.isProductionServer()
+       && AppInventorFeatures.enableHttpRedirect()) {
+        String qs = request.getQueryString();
+        String host = request.getServerName();
+        if (qs != null) {
+           String redirect = "https://" + host + "/?" + qs;
+           response.sendRedirect(redirect);
+        } else {
+           String redirect = "https://" + host;
+           response.sendRedirect(redirect);
+        }
+     return;
+   }
+   response.setHeader("Strict-Transport-Security", "max-age=3600");
```

Comment:
At what point should we considering increasing the HSTS value? My understanding is that the recommended best practice is a fairly long lived value (e.g., 1 year)

Reply:
For now, I want to consider the ability to back out of https. If we do, then people who have received the HSTS header from us will be OK after an hour. But if we use a 1-year value, then we are committed for sure. So for now, let's not be that committed!